src/trace/traceManager.js

```diff
@@ -122,7 +124,8 @@ TraceManager.prototype.getCallDataAt = function (stepIndex, callback) {
   if (check) {
     return callback(check, null)
   }
-  var callDataChange = traceHelper.findLowerBound(stepIndex, this.traceCache.callDataChanges)
+  var callDataChangeIndex = util.findLowerBound(stepIndex, this.traceCache.callDataChanges)
```

Comment:
It is probably worthwhile to also implement `findLowerBoundValue` inside `util` to return the value at the index or `undefined` if the index is out of range (just as the function was before).